py3status/modules/nvidia_temp.py

```diff
@@ -32,9 +36,21 @@ class Py3status:
     """
     # configuration parameters
     cache_timeout = 10
-    format_prefix = "GPU: "
-    format_units = u"°C"
-    temp_separator = '|'
+    format = "GPU: {gpu_temp}"
+    gpu_temp = u"{temp}°C"
```

Comment:
unless you want to do the conversions but again I do that as a different PR. who uses °F anyway, oh yeah...